### DISCUSSION TITLE
Question about random seed with ConservedLangevinNoise

### DISCUSSION THREAD

**Discussion by mangerij** [idaholab/moose#21379] [Q&A Modules: Phase field] [answered]
I have four calculations that I am running simultaneously on a cluster with 32 processors each (parallel across the nodes).
Each of the calculations has the same random seed given to the variables (set in the UserObject as instructed).

Can I trust that the noise that is given to my four calculations is the same because I have the same number of processors?

**Comment by hugary1995** [comment, marked as answer]
The noise should be the consistent even with different parallel setting:

> MOOSE currently distributes a high-quality efficient Pseudo Random Number Generator package (mtwist) that is stable across different machine architectures. This random number generator is tied into MOOSE's random number generator system that can generate consistent spatial random number fields as parallel discretization changes (e.g. the number of threads/processors does not impact generated fields). The random number interface is very straightforward to use.

See https://mooseframework.inl.gov/source/interfaces/RandomInterface.html

**Reply by hugary1995** [reply]
You can always confirm this by either
- outputting the generated field for visualization
- outputting some metrics about the sampled field using postprocessors
